pandas/core/series.py

```diff
@@ -1604,64 +1604,131 @@ def duplicated(self, keep='first'):
               occurrence.
             - 'last' : Mark duplicates as ``True`` except for the last
               occurrence.
-            - ``False`` : Mark all duplicates as ``True``.
+            - ``False`` : Mark all duplicates as ``True``. This option is not
+              compatible with ``return_inverse``.
+        return_inverse : boolean, default False
+            If True, also return the selection from the index from the Series
+            of unique values (created e.g. by selecting the boolean complement
+            of the first output, or by using `.drop_duplicates` with the same
+            `keep`-parameter) and how they relate to the index of the current
+            Series. This allows to reconstruct the original Series from the
+            subset of unique values, see example below.
+
+            .. versionadded:: 0.24.0
+
+        Returns
+        -------
+        duplicated : Series or or tuple of Series if return_inverse is True
+
+        Notes
+        -----
+        The `return_inverse`-keyword works as expected for
+        ``keep='first'|'last'``, but cannot be used together with
+        ``keep=False`` (since discarding all duplicates makes it impossible to
+        construct an inverse).
 
         Examples
         --------
         By default, for each set of duplicated values, the first occurrence is
         set on False and all others on True:
 
-        >>> animals = pd.Series(['lama', 'cow', 'lama', 'beetle', 'lama'])
+        >>> animals = pd.Series(['lama', 'cow', 'lama', 'beetle', 'lama'],
+                                index=[1, 4, 9, 16, 25])
```

Comment:
to get this to pass the doctests, you need to add `...` here at the beginning of a 'continued' line. See here for some details: http://pandas.pydata.org/pandas-docs/stable/contributing_docstring.html#tips-for-getting-your-examples-pass-the-doctests

Reply:
@jorisvandenbossche Thanks a lot!